beacon-chain/rpc/prysm/v1alpha1/validator/proposer_empty_block.go

```diff
@@ -35,11 +35,16 @@ func getEmptyBlock(slot primitives.Slot) (interfaces.SignedBeaconBlock, error) {
 		if err != nil {
 			return nil, status.Errorf(codes.Internal, "Could not initialize block for proposal: %v", err)
 		}
-	default:
+	case slots.ToEpoch(slot) < params.BeaconConfig().ElectraForkEpoch:
 		sBlk, err = blocks.NewSignedBeaconBlock(&ethpb.SignedBeaconBlockDeneb{Block: &ethpb.BeaconBlockDeneb{Body: &ethpb.BeaconBlockBodyDeneb{}}})
 		if err != nil {
 			return nil, status.Errorf(codes.Internal, "Could not initialize block for proposal: %v", err)
 		}
+	default:
+		sBlk, err = blocks.NewSignedBeaconBlock(&ethpb.SignedBeaconBlockElectra{Block: &ethpb.BeaconBlockElectra{Body: &ethpb.BeaconBlockBodyElectra{}}})
+		if err != nil {
+			return nil, status.Errorf(codes.Internal, "Could not initialize block for proposal: %v", err)
+		}
```

Comment:
I know this is just repeating the existing pattern, but this switch feels upside down. Typically we write this kind of type switch where the inequality is the other way:
 ```
 bEpoch := slots.ToEpoch(slot) 
 if bEpoch >= params.BeaconConfig().ElectraForkEpoch {
 ... // do electra thing
 }
 if bEpoch >= params.BeaconConfig().DenebForkEpoch {
 ... // do deneb thing
 }
 // do phase 0 thing
 ```